src/Game/GamePage.js

```diff
@@ -37,6 +54,14 @@ const GamePage = (props) => {
     return totalPlayerData.find((playerData) => playerData.name === playerName);
   }, [totalPlayerData]);
 
+  const turnPlayer = useMemo(() => {
+    return totalPlayerData.find(
+      (playerData, index) => index === currentTurnPlayer
+    );
+  }, [currentTurnPlayer, totalPlayerData]);
+
+  const focusedPlayer = turnPlayer && turnPlayer.name === playerName;
```

Comment:
`focusedPlayer` doesnt really reflect what this is doing, maybe something like `isMyPlayersTurn`